sofar/sofa.py

```diff
@@ -1389,21 +1429,18 @@ def verify(self, issue_handling="raise", mode="write"):
         # ---------------------------------------------------------------------
         # 8. check deprecations
         # (so far there are only deprecations for the convention)
-        if self.GLOBAL_SOFAConventions in \
-                deprecations["GLOBAL:SOFAConventions"]:
-            convention = self.GLOBAL_SOFAConventions
+        if status == 'deprecated':
             msg = ("Detected deprecations:\n"
                    f"- GLOBAL_SOFAConventions is "
-                   f"{self.GLOBAL_SOFAConventions}, which is deprecated. Use "
-                   "Sofa.upgrade_convention() to upgrade to "
-                   f"{deprecations['GLOBAL:SOFAConventions'][convention]}")
```

Comment:
This part of the error message failed in case a conventio is deprecated but a later version of the same convnetion is available. Simplified the error message to work in all cases